library/sketches-library/src/main/java/uk/gov/gchq/gaffer/sketches/datasketches/sampling/serialisation/ReservoirItemsSketchSerialiser.java

```diff
@@ -67,19 +67,7 @@ public boolean isConsistent() {
 
     @Override
     public boolean equals(final Object obj) {
-        if (this == obj) {
-            return true;
-        }
-
-        if (null == obj || getClass() != obj.getClass()) {
-            return false;
-        }
-
-        final ReservoirItemsSketchSerialiser serialiser = (ReservoirItemsSketchSerialiser) obj;
-
-        return new EqualsBuilder()
-                .append(arrayOfItemsSerDe, serialiser.arrayOfItemsSerDe)
-                .isEquals();
```

Comment:
Is this no longer possible to use or was it superfluous?

Reply:
I added tests for this class that weren't there before and they failed due to generics so I just made the equals method simpler. Might have been a smarter way to fix the test using generics.